File: DemoServer/AdditionalLanguages/php/textSearch/fTSQuerySearchBasics/FTSQuerySearchBasics.php
<?php

namespace RavenDB\Demo\textSearch\fTSQuerySearchBasics;

//region Usings
//endregion

use RavenDB\Demo\common\DocumentStoreHolder;
use RavenDB\Demo\common\models\Employee;

class FTSQuerySearchBasics
{
    public function __invoke(RunParams $runParams): array
    {
        $term1 = $runParams->getTerm1();
        $term2 = $runParams->getTerm2();

        //region Demo
        $employeesWithMatchingTerms = [];

        $session = DocumentStoreHolder::getStore()->openSession();
        try {
            //region Step_1
            $employeesWithMatchingTerms = $session->query(Employee::class)
            //endregion
                //region Step_2
                ->search("Notes", $term1 . " " . $term2)
                //endregion
                //region Step_3
                ->toList();
                //endregion
        } finally {
            $session->close();
        }
        //endregion
        return $employeesWithMatchingTerms;
    }
}

File: DemoServer/AdditionalLanguages/python/textSearch/fTSQuerySearchBasics/ftsQuerySearchBasics.py
from demo_example import Example, RunParamsBase
from models import Employee

class RunParams(RunParamsBase):
    def __init__(self, term1: str = None, term2: str = None):
        self.term1 = term1
        self.term2 = term2

class FtsQueryBasics(Example):
    def run(self, run_params: RunParams):
        term1 = run_params.term1
        term2 = run_params.term2

        #region Demo
        with self.document_store_holder.store().open_session() as session:
            
            #region Step_1
            my_query = session.query(object_type=Employee)
            #endregion
            #region Step_2
            my_query = my_query.search("Notes", term1 + " " + term2)
            #endregion
            
            #region Step_3
            employee_with_matching_terms = list(my_query)
            #endregion
        #endregion
        return employee_with_matching_terms
